### PR TITLE
Update ui_dev_server.js

### PR DESCRIPTION
**Pull request by acald-creator**
Fixed proxy.on as a function, Cypress test wasn't passing.

**Notes for Reviewers**

This PR fixes Cypress test that failed on this file. proxy.on wasn't written correctly as a function.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: provider-ui/ui_dev_server.js [hunk 1]
```diff
@@ -12,11 +12,9 @@ const httpProxy = require('http-proxy');
...
 const proxy = httpProxy.createProxyServer({});
 
-proxy.on('error', (res) => {
-  res.writeHead(500, {
-    'Content-Type' : 'text/plain',
-  });
-  res.end('Proxy issue via provider-ui.');
+proxy.on("error", function (err, req, res) {
+  res.writeHead(500, { "Content-Type" : "text/plain" });
+  res.end('Proxy issue vi Meshery Provider-UI');
```
Comment (leecalcote) [inline provider-ui/ui_dev_server.js:17]: I wonder if this is intended to be "via".

**Review by leecalcote — APPROVED** [review]

File: provider-ui/ui_dev_server.js (hunk 1 above)
Comment (acald-creator) [reply provider-ui/ui_dev_server.js:17]: Yes supposed to be via, but I copied and pasted this from `ui`, so it must be spelled wrong there too.

**Comment by acald-creator** [comment]
// @leecalcote should be good now

**Review by leecalcote — APPROVED** [review]
Lgtm

**Pull request merged**